core/server/api/settings.js

```diff
@@ -243,14 +243,14 @@ settings = {
         }
 
         return canEditAllSettings(object.settings, options).then(function () {
-            return localUtils.checkObject(object, docName).then(function (checkedData) {
-                options.user = self.user;
+            return localUtils.checkObject(object, docName).then((checkedData) => {
+                options.user = this.user;
```

Comment:
Whoops. What is this? `this.user` is not set anywhere. I think this is a leftover from [this commit.](https://github.com/TryGhost/Ghost/commit/c02ebb0dcfdd57f9cb0f2801751b6783631904bd)